### ISSUE TITLE
windows compatibility: project does not install

### ISSUE THREAD

**Issue by bholloway** [keystonejs/keystone-5#1262]
* windows 10.
* Node v10.13.0
* Yarn 1.16.0

I get an error initialising working copy.

```
> bolt
⚡️   bolt v0.22.6 (node v10.13.0)
🔎   [1/4] Validating project...
📦   [2/4] Installing project dependencies...
...
yarn install v1.9.4
[1/4] Resolving packages...
success Already up-to-date.
$ yarn build:build-field-types && preconstruct dev && node ./packages/build-field-types/bin.js dev  && yarn run --silent contributing-guide
yarn run v1.12.3
$ rimraf packages/build-field-types/lib && babel packages/build-field-types/src --out-dir packages/build-field-types/dist --ignore '**/__tests__/**/*','**/__mocks__/**/*'
Successfully compiled 42 files with Babel.
Done in 3.73s.
🎁 info project is valid!
🎁 success created links!
🎁 info project is valid!
🎁 success created links!
Input required
error Command failed with exit code 1.
```

Seems that the npm scripts in the root `package.json` is not windows friendly.

Please let me know if any further info is required for this bug.

I will raise a corresponding PR shortly.

**Comment by Vultraz** [comment]
I get a similar result (Windows 10).
```
yarn create keystone-app fsmsc-ks5
yarn create v1.16.0
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "create-keystone-app@0.6.1" with binaries:
      - create-keystone-app

╦╔═ ╔═╗ ╦ ╦ ╔═╗ ╔╦╗ ╔═╗ ╔╗╔ ╔═╗  ╦ ╔═╗
╠╩╗ ║╣  ╚╦╝ ╚═╗  ║  ║ ║ ║║║ ║╣   ║ ╚═╗
╩ ╩ ╚═╝  ╩  ╚═╝  ╩  ╚═╝ ╝╚╝ ╚═╝ ╚╝ ╚═╝

  √ Check .\fsmsc-ks5
  √ Create KeystoneJS Project
  × Install dependencies with Yarn
    → info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed.
Exit code: 1
Command: C:\Users\\AppData\Local\Yarn\bin\create-keystone-app
Arguments: fsmsc-ks5
Directory: C:\Users\\Documents
Output:

info Visit https://yarnpkg.com/en/docs/cli/create for documentation about this command.
```

**Comment by bholloway** [comment]
You need to monkey patch preconstruct with the same path escaping in my
last commit.

I will open an issue there soon.

Just super snowed under right now.

**Comment by bholloway** [comment]
See [preconstruct #77](https://github.com/preconstruct/preconstruct/pull/77)

**Comment by bholloway** [comment]
Fixed by #1463
